addons/im_livechat/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -37,6 +37,39 @@ registerInstancePatchModel('mail.messaging_notification_handler', 'im_livechat/s
             partner_name: partnerName,
         }));
     },
+
+    /**
+     * @private
+     * @param {integer} channelId
+     * @param {Object} data
+     * @param {string} [data.info]
+     * @param {boolean} [data.is_typing]
+     * @param {integer} [data.last_message_id]
+     * @param {integer} [data.partner_id]
+     */
+    async _handleNotificationChannel(channelId, data) {
+        if (data.info && data.info === 'close_livechat_session') {
+            return this._handleNotificationChannelCloseLivechatSession(channelId);
```

Comment:
As per other suggestion, only "data" should be forwarded to the handler (and channel id should be part of it).